x/stake/client/cli/query.go

```diff
@@ -114,6 +114,76 @@ func GetCmdQueryValidators(storeName string, cdc *codec.Codec) *cobra.Command {
 	return cmd
 }
 
+// GetCmdQueryValidatorUnbondingDelegations implements the query all unbonding delegatations from a validator command.
+func GetCmdQueryValidatorUnbondingDelegations(storeName string, cdc *codec.Codec) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "unbonding-delegations-from-validator [operator-addr]",
```

Comment:
Yeah i also think it’s a bit verbose but I’m open to suggestions

Reply:
Is it because it's all nested under one child/sub-command?

Reply:
you mean it should say `gaiacli query validator unbonding-delegations [operator-addr]` ?

Reply:
Precisely!

Reply:
Ok then @sunnya97's PR (https://github.com/cosmos/cosmos-sdk/pull/2565) should also add this convention ?

Reply:
How significant of a refactor to the CLI would be required to move all the module queries under their own subcommands?

Reply:
Well I think we had something similar at one point not long ago, right? Then we moved everything under `query`. But I fear this subcommand may grow big and unwieldy. I do propose we have subcommands for the modules.

Reply:
I agree. I think this is the optimal design. It will also read `gaiacli` (binary) `query` (verb) `staking` (object) `unbonding-delegations` (method) which is a pretty good from a mental model perspective.

Reply:
Agreed. How do you propose we go about this. I don't wanna block PRs.

Reply:
changed to `unbonding-delegations-from`

Reply:
Yup. Agree @fedekunze I'm opening an issue now